### DISCUSSION TITLE
List of issues to fix to help updating Lys DBM version for CC

### DISCUSSION THREAD

**Discussion by kerricsson** [chromiecraft/chromiecraft#6287] [Suggestions]
Good day,

In accordance with recent conversations at #addons sections at CC discord, I post list of issues that need to be fixed to help Lys (balleny) to update his DBM. His DBM version is used by majority of CC raiders (more that 500 players) use on weekly basis. 

Mag:

https://github.com/chromiecraft/chromiecraft/issues/6195
https://github.com/chromiecraft/chromiecraft/issues/5921
https://github.com/chromiecraft/chromiecraft/issues/5920
https://github.com/chromiecraft/chromiecraft/issues/5918

Gruul:

https://github.com/chromiecraft/chromiecraft/issues/6122

Kara:

https://github.com/chromiecraft/chromiecraft/issues/6173
https://github.com/chromiecraft/chromiecraft/issues/6162
https://github.com/chromiecraft/chromiecraft/issues/6161
https://github.com/chromiecraft/chromiecraft/issues/6253
https://github.com/chromiecraft/chromiecraft/issues/6155
https://github.com/chromiecraft/chromiecraft/issues/6133
https://github.com/chromiecraft/chromiecraft/issues/6132
https://github.com/chromiecraft/chromiecraft/issues/6131
https://github.com/chromiecraft/chromiecraft/issues/6238
https://github.com/chromiecraft/chromiecraft/issues/5606
https://github.com/chromiecraft/chromiecraft/issues/4892

Thank you in advance.

**Comment by Nyeriah** [comment]
Thanks for the suggestion, however, the list is outdated as most issues were addressed and suggesting content to be fixed is not a valid suggestion anyway. So I'll closing this one.
